interop/proto/mls_client.proto

```diff
@@ -35,10 +35,10 @@ service MLSClient {
   rpc RemoveProposal(RemoveProposalRequest) returns (ProposalResponse) {}
   rpc PSKProposal(PSKProposalRequest) returns (ProposalResponse) {}
   rpc ReInitProposal(ReInitProposalRequest) returns (ProposalResponse) {}
-  rpc AppAckProposal(AppAckProposalRequest) returns (ProposalResponse) {}
   
   rpc Commit(CommitRequest) returns (CommitResponse) {}
   rpc HandleCommit(HandleCommitRequest) returns (HandleCommitResponse) {}
+  rpc HandlePendingCommit(HandlePendingCommitRequest) returns (HandleCommitResponse) {}
```

Comment:
Right, this is meant for "pendingCommit" stacks like OpenMLS. Would it be even easier if instead of checking `sender == self` the committer immediately applied the commit they create? This means that there's no need to send a packet to the committer at all.

Reply:
The problem with the "immediately apply own commit" approach is that we might want to test that a stack behaves properly when it receives someone else's commit after having generated its own.